ckanext/unaids/theme/templates/package/read_base.html

```diff
@@ -3,10 +3,11 @@
 {% block content_action %}
   {% if not is_activity_archive %}
     {% if h.check_access('package_update', {'id':pkg.id }) %}
-        <a class="btn btn-default" href="https://navigator.unaids.org" target="_blank">
-          <i class="fa fa-compass"></i>
-          {{_('Navigator')}}
-        </a>
+      {% set navigator_locale = request.environ.CKAN_LANG.split('_')[0] %}
```

Comment:
`request.environ.CKAN_LANG.split('_')[0]` is this really the only way? It looks like a huge detour. Have you took a look at some CKAN helpers or even flask options to fetch current lang settings?

Even if it's the only way this should be extracted into a template helper (as part of `ITemplateHelpers` impl).